README.md

```diff
@@ -13,8 +13,8 @@ The scripting code uses a LabVIEW Class-based design that allows you to easily a
 
 ## Dependencies
 
-- [VeriStand 2017 or later](https://www.ni.com/ro-ro/support/downloads/software-products/download.veristand.html#382072)
-- [LabVIEW 2017 or later](https://www.ni.com/en-us/support/downloads/software-products/download.labview.html#411240)
+- [VeriStand 2019 or later](https://www.ni.com/ro-ro/support/downloads/software-products/download.veristand.html#382072)
```

Comment:
This takes me to a 2020 R5 download page. Should we change it to be 2019 R3? Or 2020 R6?

Reply:
Does letting them float to the latest version sound reasonable? I can just remove the version tags (#382072...).

Reply:
Yes, I like that idea.